src/web/components/ParticipantManagement/UpdateParticipantDialog.tsx

```diff
@@ -29,23 +34,45 @@ function UpdateParticipantDialog({
     setOpen(false);
   };
 
+  const onOpenChange = () => {
+    setOpen(!open);
+  };
+
   const originalFormValues: UpdateParticipantForm = {
     apiRoles: participant.apiRoles ? participant.apiRoles.map((apiRole) => apiRole.id) : [],
+    participantTypes: participant.types ? participant.types.map((pType) => pType.id) : [],
+    participantName: participant.name,
+    crmAgreementNumber: participant.crmAgreementNumber,
   };
 
   return (
     <Dialog
       triggerButton={triggerButton}
-      title={`Edit ${participant.name}`}
+      title={`Edit Participant: ${participant.name}`}
       closeButtonText='Cancel'
       open={open}
-      onOpenChange={setOpen}
+      onOpenChange={onOpenChange}
     >
       <Form<UpdateParticipantForm>
         onSubmit={onSubmit}
         submitButtonText='Save Participant'
         defaultValues={originalFormValues}
       >
+        <TextInput
+          inputName='participantName'
+          label='Participant Name'
+          className='text-input'
+          required
```

Comment:
let's do rules={{ required: '...' }} like the other components on the page for consistency.